manifest.json

```diff
@@ -22,14 +22,22 @@
 		"email": "win10@tutanota.com, cyp@rouquin.me"
 	},
 	"requirements": {
-        "yunohost": ">= 4.2.0"
+        "yunohost": ">= 4.2.8"
 	},
 	"multi_instance": true,
 	"services": [
 		"nginx"
 	],
 	"arguments": {
 		"install": [
+			{
+                "name": "warning",
+                "type": "display_text",
+                "ask": {
+                    "en": "/!\\ The media path has been changed to /home/yunohost.app/lufi, the files already uploaded and shared won't be downloadable anymore.",
+                    "fr": "/!\\ Le chemin des médias partagés a été changé en /home/yunohost.app/lufi, les fichiers déjà téléchargés et partagés ne seront plus téléchargeables."
+                }
+            },
```

Comment:
If it brokes previously uploaded files, it's better not to change the media path